### PR TITLE
5027 display adressess in alteration formatting

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#5027

*Description of changes:*

- Fixed formatting as UX Assurance requested
- Hid Records Office addresses for COOP business type as BAs requested

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

File: package.json
Comment (severinbeauvais) [inline package.json:3]: You also need to update (and commit) the package-lock file.

**Review by cameron-freshworks — APPROVED** [review]
Looks goods!
Was there any unit tests for the conditions at which the records address displayed?
It could be the unit tests for Addresses were skipped when we ported this component over. 
I would have a look there and update/fix any unit tests that might be applicable :)

**Review by severinbeauvais — APPROVED** [review]
GTG after comment is resolved.

File: package.json
Comment (andrepestana-aot) [reply package.json:3]: ok. Done!

**Comment by andrepestana-aot** [comment]
> Looks goods!
> Was there any unit tests for the conditions at which the records address displayed?
> It could be the unit tests for Addresses were skipped when we ported this component over.
> I would have a look there and update/fix any unit tests that might be applicable :)

No, I didn't add any for this.

**Comment by andrepestana-aot** [comment]
> GTG after comment is resolved.

Thanks!

**Review by cameron-freshworks — COMMENTED** [review]
Something is up with the package-lock.json, causing the install to fail.
I think* the only diff should be the version bump after an `npm i`, should have a look at that before merge 👍

File: package.json
Comment (severinbeauvais) [reply package.json:3]: @andrepestana-aot Let the person who created a conversation resolve it -- in my case it's my checklist to verify that the thing I commented on is fixed.

If you want to indicate that you're read or handled this thing, you can add a 👍  to the comment.

Comment (andrepestana-aot) [reply package.json:3]: ok

**Review by jachurchill — APPROVED** [review]

**Comment by andrepestana-aot** [comment]
> Something is up with the package-lock.json, causing the install to fail.
> I think* the only diff should be the version bump after an `npm i`, should have a look at that before merge 👍

@cameron-freshworks  Yes, you are right. Thanks! Could you publish it on dev, please?

**Pull request merged**
